src/views/partials/upload_evidence/uploadWamitabQualificationDescription.html

```diff
@@ -0,0 +1,10 @@
+<p>
```

Comment:
Needs IDs adding to these elements to facilitate unit / system testing. Are there unit tests to check for the existence of these elements?

Reply:
Done